.gitlab-ci.yml

```diff
@@ -0,0 +1,106 @@
+include:
```

Comment:
Maybe this is me being feature happy, but would it make sense to turn this unit of work into a template, to re-use across the spinnaker (and potentially other spring services)? Much of this seems like it could be boilerplate? Specifically the gradle tasks.

Reply:
my plan was to get it working well end to end for two of the components (preferable clouddriver and deck) and then figure out what is and isn't entirely common and extract that into a single spinnaker thing.

very much agree that our final product shouldn't be each of the 23 spinnaker services having a 95% copy paste gitlab pipeline, just wanted to have 2 concrete working impls before pulling out a lib